mllib/src/main/scala/org/apache/spark/ml/evaluation/BinaryClassificationEvaluator.scala

```diff
@@ -31,15 +31,18 @@ import org.apache.spark.sql.types.DoubleType
  * Evaluator for binary classification, which expects two input columns: rawPrediction and label.
  */
 @Experimental
+@Since("1.2.0")
 class BinaryClassificationEvaluator(override val uid: String)
```

Comment:
i tried with following comand 
spark@master:~/sparktijo/spark$ git show v1.4.0:mllib/src/main/scala/org/apache/spark/ml/evaluation/BinaryClassificationEvaluator.scala | grep  "def this() = this"
this shows the result as matching  but with 1.2.0 it is not giving any result for the above command.  Could you please  confirm.



  def this() = this(Identifiable.randomUID("binEval"))